src/pages/ItemList.js

```diff
@@ -45,67 +45,93 @@ function ItemList({ token }) {
     fetchDocs(token);
   }, [token]);
 
-  useEffect(() => {
```

Comment:
If we get rid of this, we can get rid of the `const [now, setNow] = useState(Date.now());` line and just make 
```javascript
const wasCheckedInLast24Hours =
              now - doc.last_purchased_date < ONE_DAY;
```

into 

```javascript
const wasCheckedInLast24Hours =
              Date.now() - doc.last_purchased_date < ONE_DAY;
```